Documentation/git-maintenance.txt

```diff
@@ -158,6 +159,13 @@ pack-refs::
 	need to iterate across many references. See linkgit:git-pack-refs[1]
 	for more information.
 
+vfs-cache-move::
+	The `vfs-cache-move` task operates only Scalar or VFS for Git
```

Comment:
I think the word "on" is missing between "only" and "Scalar", not that that matters a lot...